front/src/modules/settings/components/SaveAndCancelButtons/CancelButton.tsx

```diff
@@ -0,0 +1,11 @@
+import { Button } from '@/ui/input/button/components/Button';
+
+type CancelButtonProps = {
+  onCancel?: () => void;
+};
+
+export const CancelButton = ({ onCancel }: CancelButtonProps) => {
+  return (
+    <Button title="Cancel" variant="tertiary" onClick={onCancel} size="small" />
```

Comment:
According to Figma, this should be a `LightButton` with accent "tertiary" instead of a `Button`.